src/prefect/agent/kubernetes/deployment.yaml

```diff
@@ -35,11 +35,23 @@ spec:
             limits:
               memory: "128Mi"
               cpu: "100m"
+          livenessProbe:
+            exec:
+              command:
+                - python
+                - -c
+                - "from prefect.agent.kubernetes.agent import check_heartbeat; check_heartbeat()"
+            initialDelaySeconds: 40
+            periodSeconds: 40
+            failureThreshold: 2
```

Comment:
Does this mean it will take 80 seconds to actually fail?

Reply:
Yes because I think having it miss a single heartbeat isn't worthy enough to kill the process (e.g. long list of flow runs queried, updated, and deployed _could_ take a bit but definitely wouldn't take 80s)

Reply:
okie dokie, I think that's fine; we'd be looking at roughly 1.5 minute delay which is acceptable IMO

Reply:
Yeah it'll only affect performance in a really specific edge case but otherwise shouldn't be an issue (edge case is where your flow is trying to run during the first 80s that the process somehow freezes)